partiql-ast/src/main/java/org/partiql/ast/AstEnum.java

```diff
@@ -3,11 +3,19 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * TODO docs, equals, hashcode
+ * Base abstract class representing enums in the AST.
```

Comment:
```suggestion
 * Abstract base class representing enums in the AST.
```

Reply:
Ack, I'll change in a few other places as well.